css/store.css

```diff
@@ -0,0 +1,92 @@
+body {
+    margin: 0;
+}
+
+
+.section {
+    display: flex;
+    flex-direction: row;
+}
+
+
+.img--div {
```

Comment:
love the use of `--`, however, I would refrain from using the tag name as a class name. The reason being, that this class may not be applied to a div, it could be applied to any other HTML tags... If you suddenly change the tag from div to fieldset, you don't need to change the class name to img--fieldset anymore